dashboard/app/helpers/users_helper.rb

```diff
@@ -272,64 +267,90 @@ def script_progress_for_users(users, unit)
     paired_user_levels:,
     include_timestamp: false
   )
-    progress = {}
+    progress = multi_merge_user_progress_by_level(
+      script: script,
+      users: [user],
+      user_levels_by_level: {user.id => user_levels_by_level},
+      teacher_feedback_by_level: {user.id => teacher_feedback_by_level},
+      paired_user_levels: {user.id => paired_user_levels},
+      include_timestamp: include_timestamp
+    )
+
+    progress[user.id]
+  end
+
+  private def multi_merge_user_progress_by_level(
+    script:,
+    users:,
+    user_levels_by_level:,
+    teacher_feedback_by_level:,
+    paired_user_levels:,
+    include_timestamp: false
```

Comment:
```suggestion
    include_timestamp:
```
nit: will never be nil

Reply:
As is now, sure. I think it's slightly better to leave this default just in case another caller emerges eventually.